tests/query/parser/test_query.py

```diff
@@ -13,43 +13,43 @@
     SubscriptableReference,
 )
 from snuba.query.logical import OrderBy, OrderByDirection, Query
-from snuba.query.parser import parse_query
+from snuba.query.parser import build_query
```

Comment:
Most test changes from now on are because we were using fake column names on real datasets. Now they are resolved and validated so I had to fix all of them.